src/main.cpp

```diff
@@ -66,11 +59,13 @@ int main(int argc, char* argv[]){
         PROBLEM::setEoS(sph_system);
     }
     
+	createOutputDirectory(PROBLEM::output_directory);
```

Comment:
This can then move up again. That would also be necessary in case the problem setup wants to write something in the output directory.